code/modules/economy/cash.dm

```diff
@@ -17,50 +17,34 @@
 	var/worth = 0
 
 /obj/item/weapon/spacecash/c1
-	name = "1 credip chip"
```

Comment:
Oh, I see, weapon/spacecash does magic on it. Why these both exist, I don't know.